Dockerfile

```diff
@@ -1,5 +1,14 @@
-FROM alpine:3.3
-MAINTAINER jp@roemer.im
+FROM alpine:3.5
+
+# Components versions
+ENV GOLANG_VERSION 1.8
+ENV GOLANG_SRC_URL https://golang.org/dl/go$GOLANG_VERSION.src.tar.gz
+ENV GOLANG_SRC_SHA256 406865f587b44be7092f206d73fc1de252600b79b3cacc587b74b5ef5c623596
+
+# Branch name
+#ENV GOGS_VERSION v0.9.141
+ENV GOGS_VERSION develop
```

Comment:
This tag / branch name is used in docker/build.sh script.
To speed up the build process, only this branch / tag is cloned see docker/build.sh / line 12

git clone --single-branch **--branch ${GOGS_VERSION}** --depth 1 https://github.com/gogits/gogs ${GOPATH}/src/github.com/gogits/gogs